src/Mwi.LoanPay/Mwi.LoanPay.csproj

```diff
@@ -2,9 +2,9 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
-    <Version>1.2.1</Version>
-    <AssemblyVersion>1.2.1</AssemblyVersion>
-    <FileVersion>1.2.1</FileVersion>
+    <Version>2.0.1</Version>
```

Comment:
The version number should be 2.0.0. When you bump the major or minor version number the parts of the version behind it get set to 0.

Reply:
This is to match want what is in the dev environment, there was a minor change right after the major change.